Replace installation script with native ansible approach

poleszcz opened the pull request.

lootek reviewed (commented):
please be consistent using single or double quotes

also, please decide how we'll refer to RHEL-world package managers: dnf or yum or rpm?

poleszcz commented:
> please be consistent using single or double quotes
> 
> also, please decide how we'll refer to RHEL-world package managers: dnf or yum or rpm?

1. Changed to single quotes where applicable.
2. I've fixed one place where it wasn't consistent. Generally I'm reffering to rpm/deb when it's about evaluating which file shoud we download from s3 (pkg_type). I refer yum/dnf/apt where it comes to install already downloaded file and decide which pkg_manager should we use.
